src/main/java/frc/robot/AutoSelector.java

```diff
@@ -6,12 +6,26 @@
 
 public class AutoSelector {
 	public enum DesiredMode {
-		RED_3_BALL_A,
-		BLUE_3_BALL_A,
-		RED_3_BALL_B,
-		BLUE_3_BALL_B,
-		RED_1_BALL,
-		BLUE_1_BALL
+		RED_3_BALL(3),
+		BLUE_3_BALL(3),
+		RED_2_BALL(3),
+		BLUE_2_BALL(3),
+		RED_1_BALL(1),
+		BLUE_1_BALL(1),
+		LEAVE_TARMAC(0);
```

Comment:
Remove the LEAVE_TARMAC auto if it's not tested by end of day.

Reply:
Works.